website/docs/r/compute_region_disk.html.markdown

```diff
@@ -0,0 +1,231 @@
+  name = "my-disk"
+  image = "debian-cloud/debian-9"
+  size = 50
+  type = "pd-ssd"
+  zone = "us-central1-a"
+}
+
+resource "google_compute_snapshot" "snapdisk" {
+  name = "my-disk-snapshot"
+  source_disk = "${google_compute_disk.disk.name}"
+  zone = "us-central1-a"
+}
+
+resource "google_compute_region_disk" "regiondisk" {
+  name = "my-regional-disk"
+  snapshot = "${google_compute_snapshot.snapdisk.self_link}"
+  type = "pd-ssd"
+  region = "us-central1"
+
+  replica_zones = ["us-central1-a", "us-central1-f"]
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+
+* `name` -
+  (Required)
+  Name of the resource. Provided by the client when the resource is
+  created. The name must be 1-63 characters long, and comply with
+  RFC1035. Specifically, the name must be 1-63 characters long and match
+  the regular expression `[a-z]([-a-z0-9]*[a-z0-9])?` which means the
+  first character must be a lowercase letter, and all following
+  characters must be a dash, lowercase letter, or digit, except the last
+  character, which cannot be a dash.
+
+* `replica_zones` -
+  (Required)
+  URLs of the zones where the disk should be replicated to.
+
+- - -
+
+
+* `description` -
+  (Optional)
+  An optional description of this resource. Provide this property when
+  you create the resource.
+
+* `labels` -
+  (Optional)
+  Labels to apply to this disk.  A list of key->value pairs.
+
+* `size` -
+  (Optional)
+  Size of the persistent disk, specified in GB. You can specify this
+  field when creating a persistent disk using the sourceImage or
+  sourceSnapshot parameter, or specify it alone to create an empty
+  persistent disk.
+  If you specify this field along with sourceImage or sourceSnapshot,
+  the value of sizeGb must not be less than the size of the sourceImage
+  or the size of the snapshot.
+
+* `type` -
+  (Optional)
+  URL of the disk type resource describing which disk type to use to
+  create the disk. Provide this when creating the disk.
+
+* `region` -
+  (Optional)
+  A reference to the region where the disk resides.
+
+* `disk_encryption_key` -
+  (Optional)
+  Encrypts the disk using a customer-supplied encryption key.
+  After you encrypt a disk with a customer-supplied key, you must
+  provide the same key if you use the disk later (e.g. to create a disk
+  snapshot or an image, or to attach the disk to a virtual machine).
+  Customer-supplied encryption keys do not protect access to metadata of
+  the disk.
+  If you do not provide an encryption key when creating the disk, then
+  the disk will be encrypted using an automatically generated key and
+  you do not need to provide a key to use the disk later.  Structure is documented below.
+
+* `snapshot` -
+  (Optional)
+  The source snapshot used to create this disk. You can provide this as
+  a partial or full URL to the resource. For example, the following are
+  valid values:
+  * https://www.googleapis.com/compute/v1/projects/project/global/
```

Comment:
Only one of these is going to linkify - probably better to enclose them in `backticks`.